stripe/src/main/java/com/stripe/android/view/CardInputView.java

```diff
@@ -291,4 +280,24 @@ private void updateScrollToPosition() {
             mScrollToPostion = mScrollViewWidth;
         }
     }
+
+    private class BackUpFieldDeleteListener implements StripeEditText.DeleteEmptyListener {
```

Comment:
I know this is private, but would you mind adding some basic class documentation to just help other contributors along in understanding what it's for?